extensions/sportssync/src/live-scores-menubar.tsx

```diff
@@ -0,0 +1,321 @@
+import { Icon, MenuBarExtra, LocalStorage } from "@raycast/api";
+import { useFetch } from "@raycast/utils";
+import { useState, useEffect } from "react";
+import { getPreferenceValues } from "@raycast/api";
+
+// Get User Preferences
+
+interface preferences {
+  name: string;
+  id?: string;
+}
```

Comment:
You don't need to manually set preference types as this is autogenerated in `raycast-env.d.ts`

```suggestion
```